src/components/Comments/Comment.js

```diff
@@ -190,7 +271,7 @@ const Comment = ({
           </ButtonHightlight>
         </TouchableOpacity>
 
-        <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
+        {totalVote != 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for displaying the total vote count is updated. Now it only displays when `totalVote` is not zero. This is a good change as it avoids displaying unnecessary zeros.
```diff
- <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
+ {totalVote != 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->